chain/jsonrpc-primitives/Cargo.toml

```diff
@@ -13,22 +13,16 @@ description = "This crate hosts structures for the NEAR JSON RPC Requests, Respo
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-actix = "0.13.0"
-once_cell = "1.5.2"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 thiserror = "1.0"
-tracing = "0.1.13"
 uuid = { version = "~0.8", features = ["v4"] }
 
-near-chain-configs = { path = "../../core/chain-configs" }
-near-client-primitives = { path = "../client-primitives" }
 near-crypto = { path = "../../core/crypto" }
-near-metrics = { path = "../../core/metrics" }
 near-primitives = { path = "../../core/primitives" }
-near-network-primitives = { path = "../../chain/network-primitives" }
-near-primitives-core = { path = "../../core/primitives-core" }
+near-chain-configs = { path = "../../core/chain-configs" }
 near-rpc-error-macro = { path = "../../tools/rpctypegen/macro" }
+near-client-primitives = { path = "../client-primitives" }
```

Comment:
On the initial read, I thought we introduced new cruel dependencies `chain-configs` and `client-primitives`, but in fact, they just got shuffled a bit

Reply:
Few more PRs and they'll be gone for good.